### PR TITLE
Feature/add content to single tweet

### PR DESCRIPTION
**Pull request by Tretmax**
Add render content (img and video). Marged with branch create-single-tweet

### CODE REVIEW

**Review by timurkinsan — COMMENTED** [review]
missing ";"

**Comment by alexandrtovmach** [comment]
will keep it on hold, until #7 will be merged

File: src/components/SingleTweet.tsx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+`;
+const Author = styled.div`
+  height: 48px;
+  margin-top: 12px;
+  margin-bottom: 4px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.02em;
+`;
+const Logo = styled.img`
```
Comment (VladislavPokhyl) [inline src/components/SingleTweet.tsx]: Bootstrap has own implementation of rounded images. Usage :
```
import { Image} from 'react-bootstrap'
return(
         ` <Image roundedCircle={true}/>`

)
`

File: src/components/SingleTweetComment.tsx [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+const Count = styled.div`
+  padding: 0 4px;
+  font-size: 0.8rem;
+  color: rgb(83, 100, 113);
+  display: flex;
+  align-items: center;
+`;
+
+const SingleTweetComment = () => {
```
Comment (VladislavPokhyl) [inline src/components/SingleTweetComment.tsx]: Should've add typed props for mock/future data from DB e.g.
``` 
const SingleTweetComment = (
props:{
userNickName, 
userLogoUrl:string 
//...
}
) =>{
...
}
`

And then use in your styled components like:
```
`<NickName>@{props.userNickName}</NickName>`
`

File: package.json [hunk 3]
```diff
@@ -21,8 +21,10 @@
     "bootstrap": "^5.2.0",
     "react": "^18.2.0",
     "react-bootstrap": "^2.5.0",
+    "react-bootstrap-icons": "^1.8.4",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.34.0",
+    "react-icons": "^4.4.0",
```
Comment (ArtemLigerko) [inline package.json:27]: I think it is excessive, lets use react-bootstrap-icon only

File: src/components/SingleTweet.tsx [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+
+const SingleTweet = () => {
+  const bgBlue = "rgb(29, 155, 240, 0.1)";
+  const bgGreen = "rgb(0, 186, 124, 0.1)";
+  const bgRed = "rgb(249, 24, 128, 0.1)";
+  const Blue = "rgb(29, 155, 240)";
+  const Green = "rgb(0, 186, 124)";
+  const Red = "rgb(249, 24, 128)";
+  const content: string[] = [];
+  const RandomizerContent = () => {
```
Comment (ArtemLigerko) [inline src/components/SingleTweet.tsx]: function is not typed

File: src/components/SingleTweet.tsx [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+    const amountConent: number = Math.floor(Math.random() * 5 + 1);
+    for (let i = 0; i < amountConent; i++) {
```
Comment (ArtemLigerko) [inline src/components/SingleTweet.tsx]: better 'amountContent' (add 't')

**Comment by ArtemLigerko** [comment]
WM-Screenshots-20220818000737
I think it will be looking better when horizontal and vertical spaces between content will be equal

**Review by ArtemLigerko — APPROVED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**